module-telemetry/src/main/java/com/mapbox/maps/module/telemetry/MapTelemetryImpl.kt

```diff
@@ -3,9 +3,15 @@ package com.mapbox.maps.module.telemetry
 import android.content.Context
 import android.os.Bundle
 import androidx.annotation.VisibleForTesting
-import com.mapbox.android.telemetry.*
+import androidx.annotation.VisibleForTesting.PRIVATE
+import com.google.gson.Gson
 import com.mapbox.annotation.module.MapboxModule
 import com.mapbox.annotation.module.MapboxModuleType
+import com.mapbox.bindgen.Value
+import com.mapbox.common.*
```

Comment:
nit: maybe remove wildcard improts

Reply:
If we enforce this rule, we should probably enforce it with ktlint or detect, but will fix it for this PR anyway.